### PR TITLE
[sdg] add un regions

### PR DESCRIPTION
**Pull request by chejennifer**
- updates homepage to use same place dropdown as countries page
- update the dropdown to include regions
- get regions on app initialization

Screenshot 2023-09-08 at 1 46 59 PM

Screenshot 2023-09-08 at 1 45 53 PM

### CODE REVIEW

File: experimental/sdg-tracker/src/state/index.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ const MenuImageIcon = styled.img`
   border-radius: 0.25rem;
 `;
 
+const REGION_PLACE_TYPES = ["UNGeoRegion", "GeoRegion", "ContinentalUnion"];
```
Comment (juliawu) [inline experimental/sdg-tracker/src/state/index.ts:51]: To confirm, do we want `Continent` as part of this list? Not sure if the UNGeoRegions/GeoRegions include whole continents or if we have data for them.

**Review by juliawu — APPROVED** [review]
Nice!

File: experimental/sdg-tracker/src/state/index.ts (hunk 1 above)
Comment (chejennifer) [reply experimental/sdg-tracker/src/state/index.ts:51]: @n-h-diaz can you confirm these are the 3 regions we need?

Comment (n-h-diaz) [reply experimental/sdg-tracker/src/state/index.ts:51]: yes we do have data for 'Continent' too, but I was thinking of this as separate from regions (so yes those are the only three region types). not sure if we also want Continent, but maybe that's a question for UN

**Comment by chejennifer** [comment]
thanks for the reviews!

**Pull request merged**